frontend/tests/components/droplets/draggable-droplet-wide-tile.test.tsx

```
@@ -7,95 +7,389 @@ jest.mock("react-dnd", () => ({
   useDrop: jest.fn(),
 }));
 
-jest.mock("react-dnd-html5-backend", () => ({
-  HTML5Backend: {},
+jest.mock("@/lib/utils", () => ({
+  uppercaseFirstChar: (str: string) =>
+    str.charAt(0).toUpperCase() + str.slice(1),
+  cn: (...args: any[]) => args.filter(Boolean).join(" "),
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)